src/examples/using-with-cmake/CMakeLists.txt

```diff
@@ -73,27 +73,10 @@ cmake_minimum_required(VERSION 3.9)
 
 project(using_with_cmake)
 
-#
-# Provide default for ASCENT_DIR that works for an ascent install
-#
-if(NOT ASCENT_DIR)
-    set(ASCENT_DIR "../../..")
-endif()
-
-#
-# Check for valid Ascent install
-#
-get_filename_component(ASCENT_DIR ${ASCENT_DIR} ABSOLUTE)
-if(NOT EXISTS ${ASCENT_DIR}/lib/cmake/AscentConfig.cmake)
-    MESSAGE(FATAL_ERROR "Could not find Ascent CMake include file (${ASCENT_DIR}/lib/cmake/AscentConfig.cmake)")
-endif()
-
```

Comment:
removed since sub-ideal in a _"copy-from-here"_ example

Reply:
one issue -- the default logic allows this to work out of the box for folks that install themselves, which is valuable for learning the tool on say their laptop. 

I don't think it precludes `copy-from-here`,  it just gives you an error in the case it's not set as needed?

Reply:
or does this undermine when you set something like the cmake module path?

Reply:
Yes, the `if(NOT Ascent_DIR)` logic would overwrite an environment hint such as `CMAKE_PREFIX_PATH` or `Ascent_DIR` exports.

Reply:
Idea: I still want this to work in the install dir right after install --  how about (instead of being so strict) check

if 
`NOT ASCENT_DIR)` and `../../..//lib/cmake/ascent/AscentConfig.cmake` exists
set Ascent dir = ../../../
?

Reply:
I think we could set the `HINTS` path instead of `PATHS`, they have the sufficiently (low) priority: https://cmake.org/cmake/help/v3.18/command/find_package.html